learn/getting_started/installation.mdx

```diff
@@ -155,6 +166,106 @@ To deploy Meilisearch on a cloud service, follow one of our dedicated guides:
 - [Qovery](/learn/cookbooks/qovery)
 - [Railway](/learn/cookbooks/railway)
 
+### Installing older versions of Meilisearch
+
+<Tabs.Container labels={["cURL", "Docker", "Source", "Direct download"]}>
+
+<Tabs.Content label="cURL">
+
+View all available binaries under [GitHub assets](https://github.com/meilisearch/meilisearch/releases).
+
+```bash
+# Replace {meilisearch_version} and {meilisearch_os} with the specific version and OS you want to download
+# For example, if want to download v1.0 on macOS, replace {meilisearch_version} and {meilisearch_os} with v1.0 and meilisearch-macos-amd64 respectively
```

Comment:
>Won't it be annoying for a user who copies the code with these comments?

The user will need to replace `{meilisearch_version}` and `{meilisearch_os}` so copy-pasting the code won't work.

>Wouldn't this information belong outside the code sample?
It could but it would be inconsistent as we have these comments inside the code block everywhere else

Reply:
I feel some of these comments might be a bit too long, but it's your call.

(Since tone is difficult in written messages: this is not intended as passive-aggressive.)

Reply:
Oh, I think we're missing a "you" here:

> For example, if [you] want to download […]